pyvista/core/filters.py

```diff
@@ -2177,10 +2177,13 @@ def compute_cell_quality(dataset, quality_measure='scaled_jacobian', null_value=
         alg.Update()
         return _get_output(alg)
 
-    def compute_gradient(dataset, scalars=None, gradient_name='gradient',
+    @wraps(compute_derivative)
```

Comment:
We actually don't want to use this `wraps` decorator here because the arguments are/should be different.

But awesome job that you saw we use that in a lot of places!

Reply:
Gotcha. I honestly couldn't figure what `@wraps` did, but it was used in the `wireframe` function you had referenced for the deprecation warning.

Reply:
wraps just copies the function signature (docstring and argument listing). it has no effect on the code, just the docs. 

i suppose it shouldn't actually be used for wireframe as the docstring for that needs to say that it is deprecated